bin/scala

```diff
@@ -2,4 +2,4 @@
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" >& /dev/null && pwd)/.."
 
-"$ROOT/bin/common" "$ROOT/dist/target/pack/bin/scala" "$@"
+"$ROOT/bin/common" "$ROOT/dist/target/pack/bin/scala" "--power" "$@" "--offline" "--server=false"
```

Comment:
Put the forwarded arguments last?
```suggestion
"$ROOT/bin/common" "$ROOT/dist/target/pack/bin/scala" "--power" "--offline" "--server=false" "$@"
```

Reply:
we do it this way because sub commands such as `run`, `package` (from `$@`) have to come first